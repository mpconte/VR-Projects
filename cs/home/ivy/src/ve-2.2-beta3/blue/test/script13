#
# More expression tests
#
set x 0.1
set val [expr {$x + 1}]
[stdout] writeln $val
