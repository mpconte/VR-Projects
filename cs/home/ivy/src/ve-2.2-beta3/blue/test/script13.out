1.1
result = ok
